app/models/events_user.rb

```diff
@@ -0,0 +1,4 @@
+class EventsUser < ApplicationRecord
```

Comment:
Is this supposed to be events_users.rb with class EventsUsers?  (plural users)

Reply:
Not for the models. Models are singular. Linguistic Convention tells us that models are singular due to having only one model for each table. For example, we only have one Event model, one EventsUser model, and one User model. The model represents an instance of our User, EventsUser, etc.

Reply:
Ok, still getting used to the pluralization rules. I was confused and thought it should either be events_users or event_user.

Reply:
You can actually name it "EventUser" but then you have to add ```self.table_name = "events_users"``` to the model.  I'm OK with that, but it is still weird because when you access the plural of "EventUser" it will be ```:event_users``` and not ```:events_users```.  Maybe there is a way to tell rails how you want to pluralize things if you want to look into it.

Reply:
It's fine how it is. I'm new to rails, so I wasn't sure what the convention is.

Reply:
I'm with you @appleJax, I was thinking of it as a join table so both words would be plural.  Boy, it's been a loooooong time since I've looked at schema design!  :)